core/blockchain.go

```diff
@@ -3017,7 +3040,7 @@ func (bc *BlockChain) GetExternalBlockByHashAndContext(hash common.Hash, context
 func (bc *BlockChain) GetExternalBlockTraceSet(stopHash common.Hash, newHeader *types.Header, path int) ([]*types.ExternalBlock, error) {
 	extBlocks := []*types.ExternalBlock{}
 	// get the externalBlocks
-	extNewBlock, err := bc.GetExternalBlockByHashAndContext(newHeader.Hash(), path)
+	extNewBlock, err := bc.GetExternalBlock(newHeader.Hash(), []byte{}, uint64(path))
```

Comment:
Why is location always `[0, 0]` in these calls? The `path` arg doesn't make sense without a slice location